mscore/editdrumset.cpp

```diff
@@ -568,8 +568,11 @@ void EditDrumset::load()
       while (e.readNextStartElement()) {
             if (e.name() == "museScore") {
                   if (e.attribute("version") != MSC_VERSION) {
-                        QMessageBox::critical(this, tr("Drumset too old"), tr("MuseScore cannot load this drumset file."));
-                        return;
+                        QMessageBox::StandardButton b = QMessageBox::warning(this, tr("Drumset too old"),
```

Comment:
We discussed these strings somewhere few month ago...
At least, `Drumset too old` is not a correct English sentence afaiu.

Reply:
`Drumset file too old`?

Reply:
It is not supposed to be a sentence, but a window title

Reply:
To what should it get changed then, if at all?